backend/database/migrations/4_create_users_table.ts

```diff
@@ -6,9 +6,12 @@ export default class extends BaseSchema {
   async up() {
     this.schema.createTable(this.tableName, (table) => {
       table.increments('id').notNullable()
-      table.string('full_name').nullable()
-      table.string('email', 254).notNullable().unique()
-      table.string('password').notNullable()
+      table.string('usos_id').notNullable().unique()
+      table.string('first_name').notNullable()
+      table.string('last_name').notNullable()
+      table.string('student_number').notNullable().unique()
```

Comment:
znowu będziecie dropowali tablice, że migracje edytujecie? 😭

Reply:
no a jak mamy przechowywac usos_id, bez migracji?

Reply:
nie przejmuj sie nim, po prostu placze, usos_id zostaje

Reply:
@Rei-x chodzi ci o to zeby dodac jakos migracje typu `alterTable` i tam dropnąć te kolumny i dodac usos_id?

Reply:
yessir, dokładnie o to chodzi, zmodyfikuj tabele